systems/andes_user_guide.rst

```diff
@@ -11,6 +11,27 @@ conduit for large-scale scientific discovery via pre/post processing and
 analysis of simulation data generated on Summit.  Users with accounts on Summit
 will automatically be given access to Andes.
 
+Andes will be available to users soon. All users with current accounts on Rhea will receive account on Andes.
+Allocations will start fresh on Andes, but will be the same as Rhea allocations in most cases and usage will not be combined. 
+Lots of software are updated on Andes with newer verions. We recommend users to recompile their applications before launching jobs on Andes.
```

Comment:
verions -> versions
Also would suggest rewording "Lots of software are updated" to "A large number of software packages will be updated to new versions" (or something similar)

Reply:
Thanks for updating this @hliu1300